libkbfs/bserver_remote.go

```diff
@@ -182,6 +182,26 @@ func (b *BlockServerRemote) ShouldRetryOnConnect(err error) bool {
 	return !inputCanceled
 }
 
+func makeBlockIDCombo(id BlockID, context BlockContext) keybase1.BlockIdCombo {
+	// ChargedTo is somewhat confusing when this BlockIdCombo is
+	// used in a BlockReference -- it just refers to the original
+	// creator of the block, i.e. the original user charged for
+	// the block.
+	return keybase1.BlockIdCombo{
+		BlockHash: id.String(),
+		ChargedTo: context.GetCreator(),
+	}
+}
+
+func makeBlockReference(id BlockID, context BlockContext) keybase1.BlockReference {
+	return keybase1.BlockReference{
+		Bid: makeBlockIDCombo(id, context),
+		// The actual writer to decrement quota from.
```

Comment:
`makeBlockReference` is used in archive/deletes as well, so maybe "decrement quota from" -> "modify quota for"?